src/ducks/_Table/index.js

```diff
@@ -71,7 +74,13 @@ const Table = ({
     <caption>
       <Skeleton show={isLoading} className={styles.skeleton} />
       {!isLoading && items.length === 0 && (
-        <NoDataImage className={styles.nodata} />
+        <>
+          {emptySection ? (
+            emptySection
+          ) : (
+            <NoDataImage className={styles.nodata} />
+          )}
+        </>
```

Comment:
Can be simplified to `( emptySection || <NoDataImage className={styles.nodata} /> )`
```suggestion
        ( emptySection || <NoDataImage className={styles.nodata} /> )
```